game/tests.py

```
@@ -82,18 +82,68 @@ def test_page_loads(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'Checkora')
 
+    def test_authenticated_board_renders_logout_confirmation_modal(self):
+        user = User.objects.create_user(username='boarduser', password='password123')
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)